src/unix/bsd/apple/mod.rs

```diff
@@ -1172,9 +1172,9 @@ s_no_extra_traits! {
         pub ifi_noproto: u64,
         pub ifi_recvtiming: u32,
         pub ifi_xmittiming: u32,
-        #[cfg(any(target_arch = "arm", target_arch = "x86"))]
+        #[cfg(any(target_arch = "arm", target_arch = "x86", target_pointer_width = "32"))]
```

Comment:
I think it might be better to change this to just `#[cfg(target_pointer_width = "32")]` and `#[cfg(target_pointer_width = "64")]`.

Reply:
Thanks I will change this.